Analyzer/plugins/Analyzer.cc

```diff
@@ -1587,7 +1604,7 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     globalIh_ = (dedxMObj) ?  dedxMObj->dEdx() : 0.0;
     
     //Choose of Ias definition - strips only
-    dedxSObj = dedxIas_StripOnly;
+    //dedxSObj = dedxIas_StripOnly;
```

Comment:
I think we need this, because Ias should be strips only, and w/o it it will use the one that includes pixels